blueprints/app/files/package.json

```diff
@@ -13,11 +13,12 @@
   "scripts": {
     "build": "ember build --environment=production",
     "lint": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:*:fix\"",
+    "lint:fix": "npm-run-all \"lint:fix:script -- .\"",
+    "lint:fix:script": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:*:fix -- {@}\" --",
```

Comment:
The `:script` suffix feels unnecessary to me for a script name. Also, this script seems like it should not be run from the terminal, only by Ember CLI itself (?), which makes me wonder if it belongs here?

Reply:
The `:script` suffix is just to make `lint:fix` still executable as before, I put it here since I didn't know where to put it elsewhere, where do you think it would fit better than here?